Richmond_Library_App/document.py

```diff
@@ -17,19 +17,22 @@ class Django:
         
         title = fields.TextField()
         author = fields.TextField()
-        # isbn = fields.LongField()
-        year = fields.Integer()
+        isbn = fields.TextField()
+        year = fields.IntegerField()
         publisher = fields.TextField()
-        copies = fields.Integer()
-        available = fields.Integer()
+        copies = fields.IntegerField()
+        available = fields.IntegerField()
+        image = fields.TextField()
         
         # The fields of the model you want to be indexed in Elasticsearch
         fields = [
             'title',
             'author',
+            'isbn',
             'publisher',
             'copies',
             'available',
+            'image',
```

Comment:
Changed ISBN to be a text field and add field to year. Also added isbn and image to be indexed fields. Although this might not be needed since we are using querysets now instead of just putting the results into a list